### ISSUE TITLE
GDPRHelper: Typos(?) "gpdr"

### ISSUE THREAD

**Issue by m4z** [VBGAMER45/SMFMods#1]
I suppose this is by mistake:
```
> grep -Ri gdpr GDPRHelper1.0.5/ | wc -l
49
> grep -Ri gpdr GDPRHelper1.0.5/ | wc -l
674
```

It seems to affect every variable name that is written into the DB.

Technically, it's a non-issue, but I got confused when looking for the `gdpr*` files, until I noticed nearly all of them are called `gpdr*` ("PD" instead of "DP").

Do you want this to be fixed in a future version (I can write up a PR), or is it on purpose, or you want it to stay this way for compatibility reasons?

**Comment by VBGAMER45** [comment]
Yeah I thought about fixing for  compatibility  reasons as well. I did fix it for the paid version.

**Comment by m4z** [comment]
Ah, is there a general scheme for when paid version fixes and features are merged into the free version? (Like, "whatever fixes were in paid version x.y will be in the free version x.y+1, and new features in paid x.y will be in the free version x.y+3" or so.)

By the way, is there a full list of differences between the free and paid version? The [free](https://custom.simplemachines.org/mods/index.php?mod=4183) and [paid](https://www.smfhacks.com/gdprpro.php) sites are kind of hard to compare for me, because they're sorted differently, and even for (possibly) more or less identical features, the phrasing is different.
One difference seems to be exporting to more formats (html, xml) than csv. Another seems to be multi-language support. And maybe the "Privacy Policy system" does more than in the free version.

Anyway, I'd be willing to buy the paid version if you then merge the fix into the free version. 😉

**Comment by m4z** [comment]
Would you care to comment? If I offended you in any way, I'm sorry, this was not my intention.

I'd just like to move this forward (both improving the free version, and (knowing what I'm) buying (in) the paid version).

**Comment by VBGAMER45** [comment]
Haven't had time to work on this been on vacation and just really busy at the moment.
